certified-connectors/Zoho Calendar/apiProperties.json

```diff
@@ -0,0 +1,160 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2generic",
+          "clientId": "<client_id>",
+          "scopes": [
+            "ZohoCalendar.calendar.ALL",
+            "ZohoCalendar.event.ALL",
+            "ZohoCalendar.settings.ALL",
+            "ZohoCalendar.search.READ"
+          ],
+          "redirectMode": "Global",
```

Comment:
Please change this to "GlobalPerConnector", please refer this url. https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-5-submit-your-connector-for-deployment